sources/packages/web/src/components/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -69,12 +69,9 @@ export default defineComponent({
       {} as StudentApplicationOfferingChangeRequestAPIInDTO,
     );
     const approveApplicationOfferingChangeRequest = ref({} as VForm);
-    const formModel = reactive(
-      {} as StudentApplicationOfferingChangeRequestAPIInDTO,
-    );
     const cancel = () => {
       approveApplicationOfferingChangeRequest.value.reset();
-      formModel.studentConsent = false;
+      approveApplicationOfferingChangeRequestModal.value.studentConsent = false;
```

Comment:
can you double-check if the line 73 reset is working and if we need this reset?

Reply:
Correct. The reset was redundant. Removed it 👍